handlers/validator.go

```diff
@@ -398,6 +398,11 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 		validatorPageData.CurrentBalance = vbalance.Balance
 		validatorPageData.EffectiveBalance = vbalance.EffectiveBalance
 
+		if bytes.Equal(validatorPageData.WithdrawCredentials[:1], []byte{0x01}) {
+			// validators can have 0x01 credentials even before the cappella fork
+			validatorPageData.IsWithdrawableAddress = true
+		}
```

Comment:
Technically it is not required to have this in this Go routine. But since this is related to all the things happening here, I put it here for readability reasons.